Makefile

```diff
@@ -126,17 +126,26 @@ clean_venv:  # Remove venv
 
 pip-compile:  ## synchronizes the .venv with the state of requirements.txt
 	. .venv/bin/activate && ${env} python3 -m piptools compile requirements.in
+
+pip-compile-dev:  ## synchronizes the .venv with the state of requirements.txt
+	. .venv/bin/activate && ${env} python3 -m piptools compile requirements.in
```

Comment:
Not following this one (and similar pip-*-dev replacements) - why is this no longer using the dev requirements? It's now identical to pip-compile.

Reply:
The line after this also compiles the dev requirements. It's done in two steps as a failsafe so that the mandatory requirements build. (pip tools might do it well in a one liner, but that needs diving into pip tools)